consensus/src/test_utils/mod.rs

```diff
@@ -177,6 +177,7 @@ impl TreeInserter {
             &self.signer,
             failed_authors,
         )
+        .unwrap()
```

Comment:
Can you mark this `create_block_with_qc` function as test-only via `#[cfg(test)]` to ensure it's not called anywhere else and panic?

Reply:
This is already covered by the module